### PR TITLE
Split Custom Control Template / Change Toggle Control Template

### PR DESCRIPTION
**Pull request by onesounds**
- Split Custom Expander Control code to CustomControl.xaml to resource folder from app.xaml
  - It was separated to make code management easier.
- Change Toggle Switch Style (Before: left switch, right label,  / After : left label, right switch) 
  - The shape of the control was changed to match the fluent style.
  - ComboBox, CheckBox, TextBox
- Fix wrong grid and size in some setting items

**Before**
image

**After**
image

### CODE REVIEW

File: Flow.Launcher/PriorityChangeWindow.xaml.cs [hunk 1]
```diff
@@ -62,8 +61,18 @@ private void btnDone_OnClick(object sender, RoutedEventArgs e)
 
         private void PriorityChangeWindow_Loaded(object sender, RoutedEventArgs e)
         {
-            OldPriority.Text = pluginViewModel.Priority.ToString();
+            tbAction.Text = pluginViewModel.Priority.ToString();
+            //OldPriority.Text = pluginViewModel.Priority.ToString();
```
Comment (jjw24) [inline Flow.Launcher/PriorityChangeWindow.xaml.cs:65]: remove commented out code
```suggestion
```

File: Flow.Launcher.sln [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29806.167
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
```
Comment (jjw24) [inline Flow.Launcher.sln:3]:
```suggestion
# Visual Studio Version 16
VisualStudioVersion = 16.0.29806.167
```

File: Flow.Launcher.sln [hunk 3]
```diff
@@ -163,13 +163,11 @@ Global
 		{403B57F2-1856-4FC7-8A24-36AB346B763E}.Release|x86.ActiveCfg = Release|Any CPU
 		{403B57F2-1856-4FC7-8A24-36AB346B763E}.Release|x86.Build.0 = Release|Any CPU
 		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|x64.ActiveCfg = Debug|Any CPU
```
Comment (jjw24) [inline Flow.Launcher.sln:166]: revert changes to sln
```suggestion
                {1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|x64.ActiveCfg = Debug|Any CPU
```

File: Flow.Launcher.sln [hunk 4]
```diff
@@ -163,13 +163,12 @@ Global
 		{403B57F2-1856-4FC7-8A24-36AB346B763E}.Release|x86.ActiveCfg = Release|Any CPU
 		{403B57F2-1856-4FC7-8A24-36AB346B763E}.Release|x86.Build.0 = Release|Any CPU
 		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
+                {1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
```
Comment (jjw24) [inline Flow.Launcher.sln:166]:
```suggestion
              {1EE20B48-82FB-48A2-8086-675D6DDAB4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
i reverted changes to the sln file

**Comment by onesounds** [comment]
> i reverted changes to the sln file

I'm sorry. I knew that sln had changed, but I forgot to modify it.

**Comment by jjw24** [comment]
no stress, all good

**Pull request merged**